BUILD.bazel

```diff
@@ -12,7 +12,6 @@ cc_library(
     ],
     hdrs = [
         "constants.h",
```

Comment:
FYI: proper way to do this would be to have multiple cc_libraries and each one having cc_test, this way tests are cached and changes are not triggering all the tests but only those which are required.

Can be fixed later.

Reply:
added to my TODO's ;)